### DISCUSSION TITLE
Increading maximum playback speed from 2.5x to 3.0x

### DISCUSSION THREAD

**Discussion by postempirical** [PaulWoitaschek/Voice#1334] [Ideas]
I feel some heavy users, including myself are quite comfortable with speeds even upto 3.0x
This comes in really handy while listening to audio lectures. 
It would be great if soemhow his feature gets implemented in future updates. It's not a totally new feature so I feel it still respects Voice's minimalistic design approach.

**Comment by Juul** [comment]
Seconded. Even speeds above 3.0x are sometimes useful for very slow speakers and I know at least one person with serious dyslexia who likes to listen at even higher speeds. Maybe a setting that unlocks higher speeds would be appropriate? I'd be willing to implement this but I'd want the go-ahead from the maintainer that they're willing to merge such a change.

**Comment by PaulWoitaschek** [comment]
Good news! I came up with a UX pattern that allows this while not disturbing the UX for the regular playback speeds!

https://user-images.githubusercontent.com/743462/182461272-41fdaf57-2ade-4f6e-bba7-307c2a9ce7ab.mp4
